src/crate/theme/rtd/crate/page.html

```diff
@@ -88,12 +88,6 @@
 </div>
 {% endblock container %}
 
-{# Include Furo icons as first body element #}
-{% block body -%}
-{% include "partials/icons.html" %}
-{{ super() }}
-{% endblock %}
-
```

Comment:
Not sure if it is good to remove that here, and slap it only into `sidebar-secondary.html` instead?

Reply:
moved it back

Reply:
Thanks!